### PR TITLE
yarn refresh in packages/api-explorer updates to the latest API specs

### PR DESCRIPTION
**Pull request by jkaster**
- run `yarn refresh` in packages/api-explorer
- Cleaned up some codegen script files
- Put APIX script utils in a separate file
- updated all 3 API specs for APIX bundling

If desired, we could pretty-print the JSON output when we write it to the spec folder but typically we wouldn't be looking at the fetched/converted file directly so I didn't bother, particularly when our IDEs can prettify if necessary

### CODE REVIEW

File: packages/sdk-codegen-scripts/src/legacyGenerator.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    language,
+    '-o',
+    apiPath,
+    '--enable-post-process-file',
+    options,
+  ])
+}
+
+/**
+ * generate all languages for the specified configuration
```
Comment (josephaxisa) [inline packages/sdk-codegen-scripts/src/legacyGenerator.ts:79]:
```suggestion
 * Generate all languages for the specified configuration
```

Comment (josephaxisa) [inline packages/sdk-codegen-scripts/src/legacyGenerator.ts:79]:
```suggestion
 * Generate all languages for the specified configuration
```

File: packages/sdk-codegen-scripts/src/legacyGenerator.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+/**
+ * Generate API bindings with the OpenAPI legacy code generator
+ * DEPRECATED: This is using the legacy code generator perform the generation for specific API version, configuration,
+ * and language
+ *
+ * @param {string} fileName specification file name
+ * @param {IGeneratorSpec} spec for language generator options
+ * @param {ISDKConfigProps} props SDK configuration properties
+ * @returns {Promise<Buffer | string>}
```
Comment (josephaxisa) [inline packages/sdk-codegen-scripts/src/legacyGenerator.ts:53]: I think IntelliJ is generating old style doc comments for you

File: packages/api-explorer/scripts/utils.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  try {
+    console.log(
+      `Checking if "${guid}" is registered as an OAuth application ...`
+    )
+    let app = await sdk.ok(sdk.oauth_client_app(guid))
+    console.log(`${guid} is already registered as ${app.display_name}`)
+    app = await sdk.ok(sdk.update_oauth_client_app(guid, appInfo))
+    console.log(`Updated ${guid} settings`)
+    console.debug({ app })
+    return result
```
Comment (josephaxisa) [inline packages/api-explorer/scripts/utils.ts:93]: This return might not be needed due to L103.
```suggestion
```

File: packages/sdk-codegen-scripts/src/legacyGenerator.ts (hunk 2 above)
Comment (jkaster) [reply packages/sdk-codegen-scripts/src/legacyGenerator.ts:53]: That's just code I moved from legacy.ts to legacyGenerator.ts. Comment cleanup is a batch task I'm hoping to automate, possibly with a lint fix.

**Pull request merged**